components/WhatsNewSection.vue

```diff
@@ -39,9 +30,22 @@
           :alt="feature.title"
           :title="feature.title"
           :description="feature.description"
+          :sub-description="feature.subDescription"
```

Comment:
`description` becomes `caption`
and `sub-description` becomes `description`

Reply:
Resolved at https://github.com/arcana-network/testnet-website/pull/22/commits/93581b64882442545b66492de037ab310dfd20ea